tests/php/Unit/Printer/TypePrinter/SymbolPrinterTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhelTest\Unit\Printer\TypePrinter;
+
+use Phel\Lang\Symbol;
+use Phel\Printer\TypePrinter\SymbolPrinter;
+use PHPUnit\Framework\TestCase;
+
+final class SymbolPrinterTest extends TestCase
+{
+    /**
+     * @dataProvider printerDataProvider
+     */
+    public function testPrint(string $expected, Symbol $symbol): void
+    {
+        self::assertSame(
+            $expected,
+            (new SymbolPrinter())->print($symbol)
+        );
+    }
+
+    public function printerDataProvider(): \Generator
+    {
+        yield [
+            'expected' => 'name',
+            'symbol' => Symbol::createForNamespace('ns/test', 'name'),
+        ];
+        yield [
+            'expected' => 'name',
+            'symbol' => Symbol::createForNamespace(null, 'name'),
+        ];
+        yield [
+            'expected' => 'name',
+            'symbol' => Symbol::create('name'),
+        ];
+        yield [
+            'expected' => '',
```

Comment:
Why is this empty? Shouldn't this be `\\?#__\|\/`

Reply:
Because the named constructor does a `strpos($name, '/');` (name in this case is the `\\?#__\|\/` string). The idea is to split the string for "removing" the namespace.
This particular example was not so good as the delimiter '/' was in the last position, that's why the `expected` var was empty. Test updated :)